tests/observability/metrics/test_vms_metrics.py

```diff
@@ -320,13 +322,50 @@ class TestVmiFileSystemMetrics:
         ],
         indirect=["file_system_metric_mountpoints_existence"],
     )
-    def test_metric_kubevirt_vmi_filesystem_capacity_used_bytes(
-        self, prometheus, vm_for_test, file_system_metric_mountpoints_existence, dfs_info, capacity_or_used
+    def test_metric_kubevirt_vmi_filesystem_capacity_used_bytes_linux(
+        self,
+        prometheus,
+        vm_for_test,
+        file_system_metric_mountpoints_existence,
+        disk_file_system_info_linux,
+        capacity_or_used,
     ):
         compare_metric_file_system_values_with_vm_file_system_values(
             prometheus=prometheus,
             vm_for_test=vm_for_test,
-            mount_point=list(dfs_info.keys())[0],
+            mount_point=list(disk_file_system_info_linux.keys())[0],
```

Comment:
missed it before:
```suggestion
            mount_point=[*disk_file_system_info_linux][0],
```
[*<dict>] gets you a list of dict keys